test/pytest/test_bridge.py

```diff
@@ -443,20 +442,17 @@ async def test_fsread1_size_hint_absent_char_device(transport):
 
 
 @pytest.mark.asyncio
-async def test_fslist1_no_watch(transport):
-    tempdir = tempfile.TemporaryDirectory()
-    dir_path = Path(tempdir.name)
-
+async def test_fslist1_no_watch(transport, tmp_path):
```

Comment:
Can you type-annotate this while you're at it?

Reply:
Done, also to all other tests (in separate commit)